src/api.worker.ts

```diff
@@ -0,0 +1,13 @@
+import { Bridge } from "@bonfire-xmpp/apibridge";
+import core, { API, Events } from "@bonfire-xmpp/core";
+
+const ctx = self as unknown as Worker;
```

Comment:
If this is just to avoid a typescript error, `self` should be defined in the relevant ambient .d.ts